### ISSUE TITLE
Problems on Windows 10

### ISSUE THREAD

**Issue by pedrodj** [Re4son/WSL-Kali-X#9]
pedro@DESKTOP-DDJ9GB4:~$  ~/start-xfce
OPERAZIONE RIUSCITA: il processo con PID 12656 (processo figlio del
PID 9880) è stato terminato.
Welcome to the VcXsrv X Server
Vendor: The VcXsrv Project
Release: 1.19.6.0

OS: Windows NT 6.2 build 9200 (64-bit)
Contact: marha@users.sourceforge.net

LoadPreferences: C:\Users\Pedro\AppData\Roaming\.XWinrc not found
LoadPreferences: Loading C:\Program Files\VcXsrv\system.XWinrc
Warning: Locale not supported by X, falling back to 'C' locale.
(II) AIGLX: Testing pixelFormatIndex 1
(II) AIGLX: enabled GLX_SGI_make_current_read
(II) AIGLX: enabled GLX_MESA_copy_sub_buffer
(II) AIGLX: enabled GLX_SGI_swap_control and GLX_MESA_swap_control
(II) AIGLX: enabled GLX_SGIX_pbuffer
(II) AIGLX: enabled GLX_ARB_multisample and GLX_SGIS_multisample
(II) 103 pixel formats reported by wglGetPixelFormatAttribivARB
(II) GLX: Initialized Win32 native WGL GL provider for screen 0
winClipboardThreadProc - DISPLAY=127.0.0.1:0.0
OS maintains clipboard viewer chain: yes
[ ok ] Stopping system message bus: dbus.
[ ok ] Starting system message bus: dbus.
sudo: killall: command not found
touch: cannot touch '.Xdefaults': Permission denied
xrdb: Can't open display ''
session_init(): Can't open display.
/usr/bin/startxfce4: Starting X server

config/udev: failed to bind the udev monitor
[config] failed to pre-init udev

X.Org X Server 1.20.4
X Protocol Version 11, Revision 0
Build Operating System: Linux 4.9.0-8-amd64 x86_64 Debian
Current Operating System: Linux DESKTOP-DDJ9GB4 4.4.0-17134-Microsoft #706-Microsoft Mon Apr 01 18:13:00 PST 2019 x86_64
Kernel command line: BOOT_IMAGE=/kernel init=/init ro
Build Date: 05 March 2019  08:11:12PM
xorg-server 2:1.20.4-1 (https://www.debian.org/support)
Current version of pixman: 0.36.0
        Before reporting problems, check http://wiki.x.org
        to make sure that you have the latest version.
Markers: (--) probed, (**) from config file, (==) default setting,
        (++) from command line, (!!) notice, (II) informational,
        (WW) warning, (EE) error, (NI) not implemented, (??) unknown.
(==) Log file: "/var/log/Xorg.0.log", Time: Thu Jun  6 12:55:41 2019
(==) Using system config directory "/usr/share/X11/xorg.conf.d"
(EE)
Fatal server error:
(EE) parse_vt_settings: Cannot open /dev/tty0 (Input/output error)
(EE)
(EE)
Please consult the The X.Org Foundation support
         at http://wiki.x.org
 for help.
(EE) Please also check the log file at "/var/log/Xorg.0.log" for additional information.
(EE)
(EE) Server terminated with error (1). Closing log file.
xinit: XIO:  fatal IO error 11 (Resource temporarily unavailable) on X server ":0"
connection to X server lost      after 7 requests (7 known processed) with 0 events remaining.

[ ok ] Stopping system message bus: dbus.

Any suggestion?

Thank you :)

**Comment by diveyez** [comment]
Microsoft Nuked This Functionality

**Comment by ThomasArmstrongJr1247** [comment]
Diveyez, so your saying no need for me to even try it on the newest build? Just trying to save myself some headache.

**Comment by diveyez** [comment]
Yes, do not bother. Just run with a Virtual Machine if you need to have a desktop and all that.
